### DISCUSSION TITLE
Can I watch early version replay？

### DISCUSSION THREAD

**Discussion by jk9746987** [fraxiinus/ReplayBook#191] [Feature Request]
Hi, my English is not ok.
if you read this discuss to difficulty,
Sorry for the inconvenience caused.

I want watch early version replay
how to do this

**Comment by fraxiinus** [comment]
I will try to describe as best as I can through pictures.

1. Find the replay you want to play, look at the patch number, this one is "Patch 12.7"

    ![image](https://user-images.githubusercontent.com/8365233/172966927-8ae278b6-8ff1-4c8b-bfa8-28892b4f6d25.png)

2. Go to https://replays.xyz/old-clients and download the patch number that you want.

    ![image](https://user-images.githubusercontent.com/8365233/172967008-c14017a1-7a2f-4a7e-bc87-b5bf3f5c21e4.png)

3. Extract it using [7zip](https://www.7-zip.org/) and use the password shown

    ![image](https://user-images.githubusercontent.com/8365233/172967251-afe00b85-ec74-49de-aba2-a6dd74194c77.png)

4. Add the folder to ReplayBook

    ![image](https://user-images.githubusercontent.com/8365233/172967445-06953e3c-a9b2-4279-a3a2-48bbe40bb154.png)

**Reply by jk9746987** [reply]
Wow you are the sweetest.
I tried to find early version file yesterday, but no succeed.
Thank you very much.
When I back home, I will try this way.
